src/transac/common/common.module.coffee

```diff
@@ -0,0 +1,7 @@
+###
+#   @desc The Common module is the container reference for all application specific components
+###
+angular.module('transac.common',
+  [
+    #deps
```

Comment:
It is a placeholder following Todd Motto's file structure, I sometimes feel leaving this means if other developers want to add something, it may act as a hint.. But I'm not sure. Should I remove it?